dockers/docker-sonic-mgmt-framework/rest-server.sh

```diff
@@ -2,27 +2,31 @@
 
 # Startup script for SONiC Management REST Server
 
-# Read basic server settings from REST_SERVER|default entry
-HAS_REST_CONFIG=$(sonic-cfggen -d -v "1 if REST_SERVER and REST_SERVER['default']")
-if [ "$HAS_REST_CONFIG" == "1" ]; then
-    SERVER_PORT=$(sonic-cfggen -d -v "REST_SERVER['default']['port']")
-    CLIENT_AUTH=$(sonic-cfggen -d -v "REST_SERVER['default']['client_auth']")
-    LOG_LEVEL=$(sonic-cfggen -d -v "REST_SERVER['default']['log_level']")
-
-    SERVER_CRT=$(sonic-cfggen -d -v "REST_SERVER['default']['server_crt']")
-    SERVER_KEY=$(sonic-cfggen -d -v "REST_SERVER['default']['server_key']")
-    CA_CRT=$(sonic-cfggen -d -v "REST_SERVER['default']['ca_crt']")
+# Read basic server settings from mgmt vars entries
+MGMT_VARS=$(sonic-cfggen -d -t /usr/bin/mgmt_vars.j2)
```

Comment:
>sonic-cfggen -d -t /usr/bin/mgmt_vars.j2 [](start = 12, length = 40)

Could you add error handling for this command or the whole script?

Dev could make mistakes on file path, j2 content. Or Redis server crashes.

Reply:
What is recovery mechanism if either of those errors happens? Currently these errors appears in syslog as sonic-cfggen would throw. I had a bug where I oversaw the syslog and checked the docker for container of interest to be running.

Should we exit with failure so the container would not start if template is not present?